messageapi/src/main/java/fi/hel/verkkokauppa/message/dto/MessageDto.java

```diff
@@ -31,15 +33,19 @@
     @NotBlank
     private String body;
 
-    public MessageDto(String orderId, String receiver, String header, String body) {
-        this.orderId = orderId;
+    @NotNull
```

Comment:
Does this prevent sending email without attachements?

Reply:
One can send an empty map